vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -510,6 +515,19 @@
       "skip_clean": true,
       "pause_after": "60s"
     },
+    {
+      "type": "shell",
+      "inline": [
+        "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL={{user `sgx_install`}} IMG_SKU={{user `img_sku`}} /bin/bash -ux /home/packer/install-fips-dependencies.sh"
+      ]
+    },
+    {
```

Comment:
ahh I see now why you wanted to avoid this - this'll add a fair bit of latency to non-FIPS builds for no reason

Reply:
could potentially create new template(s) for FIPS specifically, though that has disadvantages of its own...

wonder what @alexeldeib thinks

Reply:
+1 to https://github.com/Azure/AgentBaker/pull/3205#discussion_r1196780672

avoid the separate reboot. do it inside the script. have the script no-op if it's not fips.

Reply:
did the UA install in pre-install, back to the original number of reboots